server/src/main/java/eu/fasten/server/plugins/kafka/FastenKafkaPlugin.java

```diff
@@ -176,6 +199,35 @@ private void emitMessage(KafkaProducer<String, String> producer, String topic, S
         producer.flush();
     }
 
+    /**
+     * Writes {@link RevisionCallGraph} to JSON file and return JSON object containing
+     * a link to to written file.
+     *
+     * @param result String of JSON representation of {@link RevisionCallGraph}
+     * @return Path to a newly written JSON file
+     */
+    private String writeToFile(String result)
+            throws IOException, NullPointerException {
+        var path = plugin.getOutputPath();
+        var pathWithoutFilename = path.substring(0, path.lastIndexOf("/"));
```

Comment:
You are assuming a unix filesystem here; this will probably not work on windows. Please make cross-platform

Reply:
Replaced all "/" that I could find with File.separator, so now it should work on every platform.